playwright.config.ts

```diff
@@ -0,0 +1,109 @@
+import type { PlaywrightTestConfig } from '@playwright/test'
```

Comment:
This file now exists at django-ory-cloud/playwright.config.ts. Please delete this file.

Reply:
that means we have to have a config in each example? but makes sense if every example folder is a "minirepo" basically.

Reply:
That's exactly what a mono-repo is: a collection of independent code bases that are all in one Git repository. This monorepo setup has many advantages that I outline in https://kevingoslar.medium.com/monorepos-make-inner-source-come-to-life-bd1592b0cadf.

Anyways, we got two copies of this file right now. One of them has to go. If you want to keep the one in the root folder as a shared config for all other codebases, that's fine with me too. Then please delete the other one in `django-ory-cloud`.